fcl/dunefdvd/gen/solar/prodmarley_flat_dunevd10kt_1x8x6_3view_30deg.fcl

```diff
@@ -0,0 +1,5 @@
+#include "prodmarley_nue_spectrum_dunevd10kt_1x8x6_3view_30deg.fcl"
```

Comment:
Same comment as above

Reply:
Now this one inherits from the 1x8x6 file.